### PR TITLE
feat: emerauthile auto generation

### PR DESCRIPTION
**Pull request by lanjoni**
# Feat for auto generation

Auto generation file has been implemented with a new model of YAML generated.

### CODE REVIEW

**Review by J0sueTM — APPROVED** [review]

**Comment by gbiz0** [comment]
LGMT

File: config.go [hunk 1]
```diff
@@ -6,6 +6,83 @@ import (
...
+          - /auth/*
+          - /home
+      - id: store
+        contents:
+          - /store/*
+      - id: marketplace
+        contents:
+          - /marketplace/*
+  # usually the only access a user could get to your services would
+  # be trough the front, but lets suppose you also have a public api.
```
Comment (grigolin) [inline config.go:32]: Fix typo: Should be "through" instead

File: docs/emerauthile.example.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          - /auth/*
+          - /home
+      - id: store
+        contents:
+          - /store/*
+      - id: marketplace
+        contents:
+          - /marketplace/*
+  # usually the only access a user could get to your services would
+  # be trough the front, but lets suppose you also have a public api.
```
Comment (grigolin) [inline docs/emerauthile.example.yaml:23]: Fix typo: Should be "through" instead

**Review by Ujs74wiop6 — COMMENTED** [review]
LGTM

**Review by gbiz0 — APPROVED** [review]
LGTM

**Review by grigolin — APPROVED** [review]
LGTM

**Review by Ujs74wiop6 — APPROVED** [review]

**Pull request merged**